package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lambdatest-cypress-cli",
-  "version": "2.0.9",
+  "version": "2.0.10",
```

Comment:
@japneetlambdatest can we make this 3.0.0 after 2.0.9, else we will have to take it till 2.0.99 😄

Reply:
Let's wait for a major release for that :P 
A lot of new changes planned in v1.2